python/cuml/dask/ensemble/base.py

```diff
@@ -130,6 +147,8 @@ def _concat_treelite_models(self):
     def _predict_using_fil(self, X, delayed, **kwargs):
         data = DistributedDataHandler.create(X, client=self.client)
         self.datatype = data.datatype
+        if self.local_model is None:

```

Comment:
I'm not totally sure how this worked before? Is there a chance that led to this need or was it just a bonus bugfix added in this PR?

Reply:
This is an additional clean up. This step was originally performed in the respective dask RF files. Now, the dask RF code is updated such that the `predict_using_fil` function is only present in and directly called from the `base.py` file